workspace/compression_sandbox/cedrlang/cedrlang.py

```
@@ -432,14 +432,16 @@ def compile_doc(text: str) -> str:
 
     for line in lines:
-        stripped = line.strip()
-        if stripped.startswith("```"):
+        if "`" in line and line.strip().startswith("```"):
             in_fenced_code = not in_fenced_code
             compiled_lines.append(line)
         elif in_fenced_code:
             compiled_lines.append(line)
         else:
-            compiled_lines.append(translate_line(line, to_compressed=True))
+            if not COMP_SINGLE_REGEX.search(line):
```

Ruff (0.16.2)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)